src/services/user.services.ts

```diff
@@ -243,3 +243,36 @@ export const putLoomieTeam = async (
     return [null, true];
   }
 };
+
+export const fuseLoomies = async (
```

Comment:
You are not retrying the request when the response is a 401 (Access Token expired).